broker/src/test/java/org/apache/rocketmq/broker/topic/TopicValidatorTest.java

```diff
@@ -70,6 +64,62 @@ public void testAddSystemTopic() {
         assertThat(TopicValidator.getSystemTopicSet()).contains(topic);
     }
 
+    @Test
+    public void testValidateSystemTopic() {
+        RemotingCommand response = RemotingCommand.createResponseCommand(-1, "");
+
+        String topic = TopicValidator.RMQ_SYS_SCHEDULE_TOPIC;
+        Boolean res = TopicValidator.validateSystemTopic(topic, response);
+        assertThat(res).isFalse();
```

Comment:
1. validateSystemTopic -> isSystemTopic ?
2. RMQ_SYS_SCHEDULE_TOPIC is not system topic? why return false here.